### ISSUE TITLE
2 banner landmark rules triggered with div[role="main"]

### ISSUE THREAD

**Issue by nickirios** [dequelabs/axe-core#1996]
I ran the axe chrome extension (version info below) on a page that has a structure like the following.
```
<header></header>
<div role="main">
  ...
  <header class="clp-header"></header>
  ...
</div>
<footer></footer>
```

The "Banner landmark must not be contained in another landmark" and "Document must not have more than one banner landmark" rules were both triggered.

![must-not-be-in-another-landmark](https://user-images.githubusercontent.com/1103623/72938224-6e2dd780-3d1f-11ea-8c91-580f64d8476d.png)

![more-than-one-banner](https://user-images.githubusercontent.com/1103623/72938231-70903180-3d1f-11ea-94c1-6d6939a2770f.png)

If I change `<div role="main">` to `<main>` and run axe again, those don't get triggered. 

I will be changing the `<div>` to `<main>`, but I would expect `<div role="main">` to not trigger those rules. It causes confusion and seems to communicate that there can't be more than one `<header>` element on a page.

## Version Info:
- axe chrome extension v4.2.0
- axe-core 3.4.0
- mac OS, chrome browser

**Comment by scurker** [comment]
Thanks for filing an issue @nickirios!

This is correct as the rule only looks at sectioning elements according to the [accessibility api mappings](https://www.w3.org/TR/html-aam-1.0/#details-id-47).

However, I can understand how this could trip people up as the [help page](https://dequeuniversity.com/rules/axe/3.4/landmark-banner-is-top-level) only mentions landmarks and landmark roles. @WilcoFiers since the guidelines specifically apply to sectioning elements and not just aria landmarks, do you think help descriptions could get tightened up?
